serverless/lambda/src/main/java/uk/gov/di/services/CodeStorageService.java

```diff
@@ -28,4 +32,14 @@ public void saveEmailCode(String email, String code, long codeExpiryTime) {
                 redisConnectionService.getValue(
                         EMAIL_KEY_PREFIX + HashHelper.hashSha256String(emailAddress)));
     }
+
+    public void deleteCodeForEmail(String emailAddress) {
```

Comment:
Worth popping a unit test on this method? To check it calls through to redis.

Reply:
Yep, I'll do that. Good shout!

Reply:
I've updated and also added a check in the VerifyCodeRequestHandler happy path test that the deleteCodeForEmail method is called.